riff-cli/README.adoc

```diff
@@ -64,8 +64,9 @@ make -C riff-cli/ build
 == Add riff binary to PATH
 ```
 cd $(go env GOPATH)/src/github.com/projectriff/riff
-cp riff-cli/riff /usr/local/bin/riff
+make -C riff-cli/ install
 ```
+NOTE: This installs the CLI in the `bin` subdirectory of the first directory listed in `$GOPATH`.
```

Comment:
I would suggest saying "This installs the CLI in $GOBIN or $GOPATH/bin"
Talking about multi-directory GOPATH is a bit confusing and not something we should recommend IMO.

Reply:
@trisberg did you want to make this NOTE consistent with the [line](#diff-7bd4a925c695c2eb0eced3872b9b965fR60) in the main README?

Reply:
Good point, fixing